### DISCUSSION TITLE
关于 antd 在externals中的配置问题

### DISCUSSION THREAD

**Discussion by my9988** [umijs/umi#6107] [Q&A]
首先我是基于antd pro v5开发的

![image](https://user-images.githubusercontent.com/17469220/106455986-f8705600-64b2-11eb-8600-5fc746f271b4.png)
官方文档有这个描述，我试着配置了 下，如下：
![image](https://user-images.githubusercontent.com/17469220/106456223-3a010100-64b3-11eb-8a18-ccd56c0b8b19.png)
build后看到antd依然在包里，有人说得改什么配置，移走按需加载，没看懂，也不知道在哪里配置，麻烦能指点下这个问题

**Comment by IsDyh01** [comment]
这个问题解决了吗哥
